dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/logger/LogcatLogHandler.kt

```diff
@@ -70,14 +70,22 @@ internal class LogcatLogHandler(
     internal fun getCallerStackElement(): StackTraceElement? {
         return if (Datadog.isDebug && useClassnameAsTag) {
             val stackTrace = Throwable().stackTrace
-            stackTrace.firstOrNull {
-                it.className !in ignoredClassNames
-            }
+            return findValidCallStackElement(stackTrace)
         } else {
             null
         }
     }
 
+    internal fun findValidCallStackElement(
+        stackTrace: Array<StackTraceElement>
+    ): StackTraceElement? {
+        return stackTrace.firstOrNull {
+            it.className !in ignoredClassNames &&
+                !it.className.startsWith("com.datadog.android.timber") &&
```

Comment:
is there any particular reason we want to exclude only `com.datadog.android.timber` packages and not the whole `com.datadog.android` package ?

Reply:
yes, having only `com.datadog.android` will break logging from the sample app.

Reply:
Ah...I see now. Maybe is better/safer this way. In this case in the alternative code I proposed above you can use `com//.datadog//.android//.timber` when generating those names or even better extract those package names in `internal static const` values and share the with the test class.